tests/e2e/utils/aws/iam.py

```diff
@@ -113,7 +113,7 @@ def delete(self):
         try:
             for policy in self.policies:
                 self.iam_client.detach_role_policy(
-                PolicyArn=policy, RoleName=self.name,
+                PolicyArn=f"arn:aws:iam::aws:policy/{policy}", RoleName=self.name,
```

Comment:
this will only work for managed policies. We should make this generic and construct policy Arn if it is managed outside of this method

Reply:
Example : https://github.com/awslabs/kubeflow-manifests/blob/main/tests/e2e/utils/ack_sm_controller_bootstrap/cleanup_sm_controller_req.py#L29-L40

Reply:
@surajkota Are you saying I should pass arns directly to this method and construct the arn in the calling method ?

Reply:
Yes